hadoop-ozone/common/dev-support/findbugsExcludeFile.xml

```diff
@@ -19,4 +19,10 @@
   <Match>
     <Package name="org.apache.hadoop.ozone.protocol.proto"/>
   </Match>
+
+  <Match>
+    <Class name="org.apache.hadoop.ozone.protocolPB.OzonePBHelper" />
+    <Method name="getFixedByteString" />
+    <Bug pattern="AT_OPERATION_SEQUENCE_ON_CONCURRENT_ABSTRACTION" />
```

Comment:
Can you please help me to understand why is it safe to ignore this warning?

Reply:
It was copied from Hadoop. Now I resolved findbug by using `putIfAbsent()`method.